packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -336,6 +369,7 @@ export const visualizeUnknownSharedObject: VisualizeSharedObject = async (
 
 /**
  * List of default visualizers included in the library.
+ * @remarks {@link DataObject} does not have type information, thus not included in the list.
```

Comment:
I think this is necessary when referencing types from other packages in TSDoc. @Josmithr can probably correct me if not.

```suggestion
 * @remarks {@link "@fluidframework/aqueduct#DataObject} does not have type information, thus not included in the list.
```

Reply:
Our tooling won't complain because this API isn't package-exported. But yes, technically the package identifier is required to be correct.